source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -899,46 +899,98 @@ Mainnet
+        - ``GetPoolInfo`` now also returns the commission rates for the current reward period.
 
-            - Support new ``invoke`` operations for retrieving account keys and checking signatures.
+        - Added a number of endpoints to the GRPCV2 API, including:
 
-        - Shut down consensus upon a protocol update updating from protocol version 6.
+            - ``GetBakersRewardPeriod``: provided a block, then it returns information about bakers for the reward period of the block.
 
-        - Fixed a bug that causes bakers in genesis to restake their earnings when they should not. This affects genesis data at protocol version P5; P1-P4 genesis data are not affected. This breaks compatibility with chains started with P5 genesis data, where some genesis bakers are not set to restake earnings. Other chains (including mainnet and testnet) are not affected.
+            - ``GetBlockCertificates``: provided a block, then it returns quorum certificate, timeout certificate and epoch finalization entry contained in the block (where present).
 
-        - Changed the ``GetConsensusStatus`` endpoint so that slot duration is only returned in protocol versions 0-5.
+            - ``GetBakerEarliestWinTime``: provided a baker ID, it returns the earliest time at which the node projects that the baker could be required to bake a block.
 
-            - Endpoint is extended to return current timeout duration, current round, current epoch and trigger block time in protocol version 6.
+            - ``GetFirstBlockEpoch``: returns the block hash of the first block in a given epoch.
 
-        - Changed the ``GetBlockInfo`` endpoint:
+            - ``GetWinningBakersEpoch``: returns a list of the bakers that won rounds in a specified (finalized) epoch. This only supports consensus version 1.
 
-            - Block slot is only returned in protocol versions 0-5.
+    **Fixes**
 
-            - In protocol version 6, the returned finalized block is the last finalized block until itself is finalized. Then it is itself.
+        - Several fixes are included for peer handling.
 
-            - Endpoint extended to return block round and epoch in protocol version 6.
+            - Fixed a bug where stored peers are removed incorrectly. The node remembers peers across restarts. When starting up it will try to connect to stored peers in addition to any supplied bootstrap and given nodes. You can use the new flag ``--clear-persisted-peers`` (environment variable ``CONCORDIUM_NODE_CLEAR_PERSISTED_PEERS``) to clear stored peers on startup. An incorrect ``peer_bucket_size`` metric calculation exposed by the bootstrapper was fixed. What was counted was not the number of peers in the bucket, but rather, roughly, how many times peers that are in the bucket have reconnected.
+
+            - Banned peers are no longer reset on startup by default. The flag ``--no-clear-bans`` has been renamed  to ``--clear-bans``; when set it will clear the banned peers on startup.
```

Comment:
This is not a "fix" either, it is a change in behaviour. The previous behaviour was as intended, we just changed what is intended.